components/company/Hero.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+  interface CompanyInfo {
+    name: string
+    description: string
+    logo: string
+    website: string
+  }
+
+  const company = defineProps({
+    content: {
+      type: Object as PropType<CompanyInfo>,
+      required: true,
+    },
+  })
+</script>
+
+<template>
+  <div>
+    <VSheet class="w-75">
+      <VRow no-gutters>
+        <VCol

```

Comment:
Need to make some sort of hover effect when hovering over the logo or title. It is not very obvious that you can click on them.